node/src/main/scala/coop/rchain/node/diagnostics/StoreMetrics.scala

```diff
@@ -28,15 +28,29 @@ object StoreMetrics extends StoreMetricsInstances {
     def g(name: String, value: Long): F[Unit] =
       m.setGauge(name, value)
 
+    def cm(name: String, value: Option[StoreUsageCount]): F[Unit] =
+      for {
+        _ <- m.setGauge(name + "-count", value.map(_.count).getOrElse(0))
+        _ <- m.setGauge(name + "-peak-rate", value.map(_.peakRate.toLong).getOrElse(0))
+        _ <- m.setGauge(name + "-current-rate", value.map(_.currentRate.toLong).getOrElse(0))
+      } yield ()
+
+    def pc(name: String, value: Option[StoreUsageCount]): F[Unit] =
```

Comment:
Is this a shorthand for _produce_ ?

Reply:
"pc" is abbreviation of produce/consume, and "cm" is for "comm".